src/utils/stringUtils.ts

```diff
@@ -15,7 +15,51 @@ class stringUtils {
      * @param replace String to replace keywords with.
      * @returns Replaced String.
      */
-    public static async customReplace(string: string, keyword: string, replace: string) {
+     public static async customReplace(string: string, keyword: string, replace: string) {
         return string.replace(new RegExp(keyword, "g"), replace);
     }
+
+    /**
+     * ### URL Parse
+     * @param string URL that want to be parsed
+     * @returns Dictionary

```

Comment:
"Dictionary" does not exist in JavaScript, in javascript they are objects.
Use proper JSDoc formatting.